SOURCES/privacyidea-pam-selinux-src/privacyidea-pam-selinux.te

```diff
@@ -0,0 +1,16 @@
+# type enforcement rules for pam_privacyidea.so
+module privacyidea-pam-selinux 1.0;
+
+require {
+    type admin_home_t;
+    type sshd_t;
+    type http_port_t;
+    class file { read getattr open };
+    class tcp_socket name_connect;
+}
+
+# Rule 1: Allow the process with type "sshd_t" to read and open the file "pam_privacyidea.so" with context "admin_home_t".
+allow sshd_t admin_home_t:file { read getattr open };
+
+# Rule 2: Allow the process with type "sshd_t" to connect to TCP port 443 (http_port_t).
+allow sshd_t http_port_t:tcp_socket name_connect;
```

Comment:
Not sure where this belongs but if we use offline mode, an sqlite-file will be created somewhere. The process might need the necessary rights as well.